palantirnet/the-build
# DEV-30 - Move phpmd config to phpmd.xml.

User story: [DEV-30: Move phpmd config into phpmd.xml](https://palantir.atlassian.net/browse/DEV-30)

### Description

This moves the phpmd config in to a separate phpmd.xml file.

### Testing instructions

1. Update the build for this branch in a project 
2. Run the build installer `vendor/bin/the-build-installer`
3. Run code review `vendor/bin/phing test` to verify that phpmd runs.
